hnn_core/network_builder.py

```diff
@@ -69,8 +70,10 @@ def _simulate_single_trial(net, tstop, dt, trial_idx):
     h.finitialize()
 
     def simulation_time():
-        print('Simulation time: {0} ms...'.format(round(h.t, 2)))
+        pbar.update(10)
```

Comment:
```suggestion
        if check_library('tqdm'):
            pbar.update(10)
        else:
             print('.', end='')
             sys.stdout.flush()
```

copy this logic into `check.py`: https://github.com/mne-tools/mne-python/blob/main/mne/utils/check.py#L105-L108